app/lib/mahonia/edtf_literal.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Mahonia
+  class EdtfLiteral < RDF::Literal
+    DATATYPE = RDF::URI('http://id.loc.gov/datatypes/edtf/EDTF')
+
+    # support ActiveJob serialization
+    include GlobalID::Identification
+
+    def self.find(id)
+      new(id)
```

Comment:
Making a new instance by default from find seems like it could cause problems. Not necessarily in the way we're currently expecting this class to be used, but just seems like something another developer using this class without having read it wouldn't expect. Just a thought.